opal/mca/accelerator/cuda/accelerator_cuda_component.c

```diff
@@ -188,7 +188,15 @@ static opal_accelerator_base_module_t* accelerator_cuda_init(void)
     if (!opal_cuda_support) {
         return NULL;
     }
-
+    int count = 0;
```

Comment:
Might as well move this line down below the `cuInit()` block (i.e., to where you know for sure that `count` will be used).